components/NewQuestion.js

```diff
@@ -94,6 +99,26 @@ export default class NewQuestion extends React.Component {
         <CardHeader sm={2}>New question</CardHeader>
         <CardBody>
```

Comment:
One thing testing locally, due to the hiding the new question panel for active course staff, if a user is on duty, they can't add someone to the queue as the panel isn't there. There will either need to be a button to reshow that panel, that feature being removed, or some other thing.